### PR TITLE
add RaghavMaskara21

### PR DESCRIPTION
**Pull request by RaghavMaskara21**
resolve issue #872

### CODE REVIEW

**Review by krishguptadev — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -1654,5 +1654,11 @@
     "githubUsername": "AnishDe12020",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/143733055-ea3a82e1-0d0d-460d-89c2-e622877974a0.png",
     "issueNumber": 870
-  }
+  },
+  {
+    "name": "Raghav Maskara",
+    "githubUsername": "RaghavMaskara21",
+    "imageUrl": "https://user-images.githubusercontent.com/79410102/143765055-791a42ab-2cc6-4c9d-9e45-6529eb147aa4.png",
```
Comment (Panquesito7) [inline data.json:1661]: I recommend more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/143782190-a08fdf59-6943-4538-874b-58276d7f7f3f.png",
```

**Comment by nhcarrigan** [comment]
Hey there @RaghavMaskara21,

We've changed the structure of the profile data, and unfortunately your pull request is no longer in line with these changes.

You'll need to rebase your branch, where you'll see the `data.json` file has been removed. Instead, you should create a file in the `profiles` directory - name the file `RaghavMaskara21.json` and add your information there.

If you run in to any issues with this, let us know!

**Comment by krishguptadev** [comment]
Hey there @RaghavMaskara21 👋 Do you need any help?

**Comment by nhcarrigan** [comment]
Hey there @RaghavMaskara21,

This PR has been inactive for a while now, so we're going to close it. You're welcome to resubmit your changes in a new PR when you are ready 🙂

**Pull request closed without merging**